Improve SEO metadata across all docs pages (#397)
powersync-ja/powersync-docs: client-sdks/frameworks/next-js.mdx

---
title: "Next.js + PowerSync"
sidebarTitle: "Next.js"

> description: "Build offline-first and local-first Next.js applications with PowerSync, including project setup, SSR considerations, and sync configuration."

Reviewer: ```suggestion
description: "Learn how to build highly responsive Next.js applications with PowerSync."
```
